site/gatsby-site/src/pages/apps/classifications.js

```diff
@@ -260,7 +260,8 @@ export default function ClassificationsDbView(props) {
       const classificationsToRowsMap = (cObj) => {
         const row = {};
 
-        row['IncidentId'] = cObj.incident_id;
+        row['IncidentId'] = cObj.incidents[0].incident_id;
```

Comment:
Is this what we wanted for the Incident Id column? Should we display just the first incident from the Classification?

Reply:
Yeah, we'll have to live with this code smell; at the moment, a CSET classification only applies to one and only one  incident.

Reply:
@kepae 
> should we throw a static build error since (a) all classifications are (currently) expected to have at least one incident, and (b) there's currently no plan for handling classifications without incidents? (per chatting yesterday)
Could also be an ERR log of some kind, for data-only problems. Build error might be too aggressive, but earlier better than later to do that.

I like that. We have a test file for this kind of check: `integrity.cy.js`; I'll add a new test.